src/benchmarkstt/metrics/core.py

```diff
@@ -188,6 +187,9 @@ def compare(self, ref: Schema, hyp: Schema):
         hyp_str = ''.join([i['item'] for i in hyp])
         total_ref = len(ref_str)
 
+        if self._mode != self.MODE_LEVENSHTEIN:
+            raise NotImplementedError('CER is only implemented for Levenshtein distance')
+
```

Comment:
When you run `benchmarkstt --help`, the entry for `--cer` says that `levenshtein` is the default mode. However, if we run a comparison with just `--cer` (without specifying a mode), for example:
```
benchmarkstt --reference-type argument --hypothesis-type argument --reference 'the quick brown fox' --hypothesis 'the quick brown fax' --cer
```
This throws the NotImplementedError because `_mode` hasn't been set, suggesting that `levenshtein` hasn't actually been set as the default.

The way to get around that would be to add the mode explicitly, e.g.
```
benchmarkstt --reference-type argument --hypothesis-type argument --reference 'the quick brown fox' --hypothesis 'the quick brown fax' --cer levenshtein
```
...but this seems pointless, since `levenshtein` is the only valid option right now.

I'd suggest changing line 183 to set `_mode` to `MODE_LEVENSHTEIN` if none is supplied, so:
```
self._mode = mode or self.MODE_LEVENSHTEIN
```

Reply:
Good point I changed the default value to Levenshtein for CER